db/compaction/subcompaction_state.h

```diff
@@ -169,6 +184,15 @@ class SubcompactionState {
     return *current_outputs_;
   }
 
+  CompactionOutputs* GetOutputs(bool is_penultimate_level) {
```

Comment:
We usually prefer not to say `Get` if accessing for mutation. Like `RangeDelAgg()` below.